### DISCUSSION TITLE
I can't update my extension, it tell me 'Extension validation error'

### DISCUSSION THREAD

**Discussion by github0null** [microsoft/vsmarketplace#546] [Feedback]
Hello !

Today i want to update my extension, but it failed, it looks like that the url `https://[vsmarketplacebadge.apphb.com](https://vsmarketplacebadge.apphb.com/version/cl.eide.svg)/version/cl.eide.svg` is invalid.

So what should i do if i want to keep svg badge ?

![image](https://user-images.githubusercontent.com/43022536/210174191-d70ff788-9975-4a47-afc0-2a8b58f7f122.png)

![image](https://user-images.githubusercontent.com/43022536/210174201-6f711899-d0e3-4e83-9e09-5586293ecc72.png)

**Comment by amit-kulkarni-in** [comment]
@github0null : Please replace with the new URL :  https://vsmarketplacebadges.dev/. Details of the issue is in this location. https://github.com/microsoft/vscode-vsce/issues/799 . Let us know if you still face any issue ..

**Reply by github0null** [reply]
Thanks !
